<?xml version="1.0"?>
		
 <launch>
 
 	<!-- including all the launchfiles for the simulation-->
 	<include file = "$(find final_assignment)/launch/simulation_gmapping.launch" />
	<include file = "$(find final_assignment)/launch/move_base.launch" />
	
	<!--my UI node-->	
	<node name = "user_interface" pkg = "final_assignment" type = "user_interface_node" output = "screen" launch-prefix="xterm -e"  required = "true" />
</launch>
